package cz.cvut.fit.stehlvo2.repository.util

class DataAccessException(throwable: Exception): RuntimeException(throwable)